modules/nodes-descheduler-policies.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-policies_{context}"]
+= Descheduler policies
+
+The following descheduler policies are available:
+
+// TODO: the example YAML probably aren't right because the operator's config is slightly different than the orig Kubernetes config.
+
+////
+Questions:
+ - refer to these as policies or strategies? scheduler says policies, but the field used here is strategies.
+ - can you configure more than one at the same time?
+    - if you can, is order important?
+ - upstream kube says they're all enabled by default, but none are enabled by default on operator?
+ - does only lownodeutilization have additional config fields? how to find out?
+////
```

Comment:
1. these are referred to as "strategies" in the descheduler code. While the scheduler uses "policies", those are technically totally different things
2. Yes, and order is not important
3. If you run the descheduler by itself all the strategies are enabled, but when you run the operator it will error out before triggering the descheduler if no strategies are configured.
4. right now only lownodeutilization has additional fields. the upstream k8s.io/descheduler repo readme are the source of truth for that (or our own openshift/descheduler readme, but that's just forked)